src/bootstrap.js

```diff
@@ -77,21 +77,21 @@ Vue.use(VueRouter);
 export const router = new VueRouter({
   routes,
 });
-router.beforeEach((route, redirect, next) => {
-  if (route.matched.some(m => m.meta.auth) && !store.state.auth.authenticated) {
+router.beforeEach((toRoute, fromRoute, next) => {
```

Comment:
I prefer to `router.beforeEach((to, from, next) => { ... });`.
So `to` instead of `toRoute` and `from` instead of `fromRoute`.